support/config/deployment.go

```diff
@@ -105,6 +106,24 @@ func (c *DeploymentConfig) ApplyTo(deployment *appsv1.Deployment) {
 	}
 	deployment.Labels[ManagedByLabel] = "control-plane-operator"
 
+	// adding annotation for EmptyDir volume safe-eviction
+	if deployment.Spec.Template.ObjectMeta.Annotations == nil {
+		deployment.Spec.Template.ObjectMeta.Annotations = make(map[string]string)
+	}
+
+	emptyDirVolumes := make([]string, 0)
+
+	for _, volume := range deployment.Spec.Template.Spec.Volumes {
+		if volume.EmptyDir != nil {
```

Comment:
[hostPath](https://kubernetes.io/docs/concepts/storage/volumes/#hostpath) is also considered a local volume, so you might want to check for that as well.

Reply:
Ohh that's true, fixed